src/coreclr/src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -601,7 +602,9 @@ internal sealed class LicenseInteropProxy
         // LicenseContext
         private readonly MethodInfo _setSavedLicenseKey;
 
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.DefaultConstructor)]
```

Comment:
Can you help me understand what `DynamicallyAccessedMembers` indicates? Seems this is a field that is populated by reflection - I thought we had to annotate the members that were accessed by reflection.

Reply:
See https://github.com/mono/linker/blob/master/docs/design/reflection-flow.md#cross-method-annotations